compiler/src/test/java/com/squareup/anvil/compiler/codegen/ContributesSubcomponentGeneratorTest.kt

```diff
@@ -515,7 +515,7 @@ class ContributesSubcomponentGeneratorTest {
       """
     ) {
       assertThat(exitCode).isError()
-      assertThat(messages).contains("Source0.kt: (11, 1)")
+      assertThat(messages).contains("Source0.kt: (16, 11)")
```

Comment:
I had to update these, because we point to the class name now instead of the annotation:
```kotlin
@Something
class Abc
```
Previously, the error pointed to `@Something`. Now it points at `Abc`.